components/colors/colors.css

```diff
@@ -68,21 +68,27 @@ html {
 
 .too-dirty {
   fill: var(--too-dirty);
+  background-color: var(--too-dirty);
```

Comment:
Either remove the fill, or add the fill only to a `svg.too-dirty, svg .too-dirty`, because this will have unexpected results down the road